frontend/src/components/InlineTextEdit.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <div>
+    <v-edit-dialog      
+      :return-value.sync="textAreaValue"
+      lazy
+      large
+      @save="$listeners.save(item, editPath, textAreaValue)"
+      @cancel="$listeners.cancel"
+      @open="open"
+    > <span :style="{padding: '4px'}">{{ item[keyPath] && item[keyPath][value] }}</span>
+      <v-textarea
+        v-if="isShowTextArea == true"
+        slot="input"
+        ref="editFolloUp"
+        v-model="textAreaValue"
+        label="Edit follow up"
+        single-line
+        counter
+      ></v-textarea>
+    </v-edit-dialog>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Prop, Vue } from "vue-property-decorator";
+
+@Component({
+  name: "InlineTextEdit"
+})
+export default class InlineTextEdit extends Vue {
+  @Prop({ default: () => ({}) })
+  item!: any;
+  @Prop() value!: any;
+  @Prop() keyPath!: string;
+  
+  isShowTextArea: boolean = false;
+  textAreaValue: string = "";
+  
+  get editPath() {
+    return `/${this.item[".key"]}/${this.keyPath}/${this.value}`;
+  }
+
+  open() {
+    // wait small amount of time so focus works
+    setTimeout(() => {
+      (this.$refs.editFolloUp as any).focus();
```

Comment:
Not only “Follow Up” can be edited by this component. Can it be more generalized?